test/e2e/config_test.go

```diff
@@ -92,6 +94,50 @@ func TestClusterMonitoringOperatorConfiguration(t *testing.T) {
 	t.Log("asserting that CMO goes back healthy after the configuration is fixed")
 	assertOperatorCondition(t, configv1.OperatorDegraded, configv1.ConditionFalse)
 	assertOperatorCondition(t, configv1.OperatorAvailable, configv1.ConditionTrue)
+
+	// Push a configuration that disables Grafana.
```

Comment:
I would extract this addition into a new independent test

Reply:
Done.